schema/pgsql/upgrades/018.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE object_id_tag (
+    object_id bytea NOT NULL REFERENCES object(id),
+    tag text NOT NULL,
+    value text NOT NULL,
+
+    CONSTRAINT pk_object_id_tag PRIMARY KEY (object_id, tag)
+);
+
+INSERT INTO object_id_tag (object_id, tag, value) SELECT id, 'host' AS tag, host FROM object;
+INSERT INTO object_id_tag (object_id, tag, value) SELECT id, 'service' AS value, service FROM object WHERE service IS NOT NULL;
```

Comment:
When giving explicit column names for the select statement, it should be `'service' AS tag` (but it should be just fine without any column aliases).